.github/workflows/jbpm_system_test_ci.yml

```diff
@@ -0,0 +1,37 @@
+# Runs jbpm-system-test module tests periodically
+
+name: jBPM System Test CI
+
+on:
+  schedule:
+    - cron:  '0 2 * * *'
+
+defaults:
+  run:
+    shell: bash
```

Comment:
FYI, according to the [docs](https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#using-a-specific-shell), `bash` is default on non-Windows platforms. But I have no issue stating that explicitly :)